geps/gep-2907/index.md

```diff
@@ -73,43 +58,34 @@ flowchart LR
+For mutual backend TLS, the gateway's identity must be configured. This identity, a client certificate set on the gateway object, will be presented by the gateway. The gateway uses the same client certificate for all backend connections originating mutual TLS.
 
-* **Terminate**: the TLS connection is instantiated between the frontend and the
-  Gateway. The connection between the Gateway and the backend is left unencrypted
- unless a new TLS connection between the two entities is configured via BackendTLSPolicy.
-* **Passthrough**: the TLS connection is instantiated between the frontend and the
-  backend. The traffic flows through the Gateway encrypted, and the Gateway is not
-  able to decrypt or inspect the encrypted portions of the TLS stream.
 
 ## Proposed Segments
-Note that this does not represent any form of commitment that any of these
-fields or concepts will exist within Gateway API. If or when they do, we propose
-the following naming structure:
 
-### 1. Validate Client Certificate provided by Frontend
+
+### 1. Gateway level configuration
+
+Gateway-level configurations are required for both Frontend and Backend connections. We propose a top-level GatewayTLSConfig that is divided into independent Backend and Frontend configurations.
 
 | Proposed Placement | Name | Status |
 |-|-|-|
-| Gateway Listener | `Listener.TLS.FrontendValidation` | Proposed |
+| Gateway | `Gateway.TLS.Frontend` | Proposed |
+| Gateway | `Gateway.TLS.Backend` | Proposed |
 
 #### Rationale
-Use FrontendValidation to leave room for concepts like trust anchors and trust
-domains. Anything not strictly tied to validation would belong to Listener.TLS
-which is now reserved exclusively for “Frontend” TLS configuration (1+2).
-
-#### Why Not Frontend.Validation?
-Part of the idea behind this GEP is that Listener.TLS is really entirely focused
-on 1+2 - the bits of TLS between frontend and Gateway. That means that if we
-were to add any additional TLS configuration in the Listener.TLS struct, it
-would tied to that limited scope, and thus we don't really need a separate
-Frontend struct.
+From the security point of view both TLS configurations require a gateway level field. The configurations are independent and can coexist.
 
-One could make an argument that Listener.TLS.Validation should be the field name
-here to avoid any ambiguity, but in this specific context, we think that it's
-probably helpful to specifically spell out frontend.
+#### Why Not Listener.Frontend?
+Due to connection coalescing, the client certificate validation for the frontend was moved to a gateway-level field to address security concerns.
```

Comment:
Recommend adding a link to relevant PR or GEP

Reply:
done